xml/System.Windows.Media.Animation/Storyboard.xml

```diff
@@ -1748,16 +1748,16 @@
         <Parameter Name="element" Type="System.Windows.DependencyObject" />
       </Parameters>
       <Docs>
-        <param name="element">The dependency object from which to get the <see cref="F:System.Windows.Media.Animation.Storyboard.TargetProperty" />.</param>
-        <summary>Retrieves the <see cref="F:System.Windows.Media.Animation.Storyboard.TargetProperty" /> value of the specified <see cref="T:System.Windows.Media.Animation.Timeline" />.</summary>
+        <param name="element">The dependency object from which to get the <xref data-throw-if-not-resolved="true" uid="System.Windows.Media.Animation.Storyboard.TargetProperty_a"></xref>.</param>
```

Comment:
I've never seen xrefs in XML - is this correct?

Reply:
This is a workaround to fix the issue. Actually cref will be transferred to xref in the downstream toolchain. However, the cref here can't be resolved correctly to the required xref link of the attached property. So I am using xref directly here, which will be ignored at downstream. It is the same case as those in CDATA.

In this way, all the links have been fixed in methods of GetTargetProperty and SetTargetProperty:
https://review.docs.microsoft.com/en-us/dotnet/api/system.windows.media.animation.storyboard.gettargetproperty?view=net-5.0&branch=pr-en-us-7205

Reply:
@carlossanlop Will having an `<xref>` in the XML cause problems for backporting these doc comments to source?